src/libServer/JSONConversion.cpp

```diff
@@ -255,6 +255,26 @@ bool JSONConversion::checkJsonTx(const Json::Value& _json) {
   return ret;
 }
 
+const vector<string> JSONConversion::convertJsonArrayToVector(
+    const Json::Value& _json) {
+  if (!_json.isArray()) {
+    throw jsonrpc::JsonRpcException(Server::RPC_INVALID_PARAMETER,
+                                    "Expected Array type");
+  }
+
+  vector<string> vec;
```

Comment:
Can you add a `using VectorOfString = std::vector<std::string>` in BaseType.h?

Reply:
This will require a major refactor as `bool Account::FetchStateJson(Json::Value& root, const string& vname,
                             const vector<string>& indices, bool temp)` is also the same.